types/twitter-text/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for twitter-text v1.14.7
+// Type definitions for twitter-text v2.0.0
 // Project: https://github.com/twitter/twitter-text
 // Definitions by: rhysd <https://rhysd.github.io>
```

Comment:
Can you update the header from `// Definitions by: rhysd <https://rhysd.github.io> ` to `// Definitions by: rhysd <https://github.com/rhysd>` so the both can identify the owner. (And optionally add yourself as an owner)

Reply:
Done 😄